build.csx

```diff
@@ -68,12 +71,31 @@ targets.Add("default", DependsOn("gitlink", "unit", "integ", "spec", "approve",
 
 targets.Add("outputDirectory", () => Directory.CreateDirectory(outputDirectory));
 
+targets.Add("coverityDirectory", () => Directory.CreateDirectory(coverityDirectory));
+
 targets.Add("logsDirectory", () => Directory.CreateDirectory(logsDirectory));
 
 targets.Add("testsDirectory", () => Directory.CreateDirectory(testsDirectory));
 
 targets.Add("build", DependsOn("clean", "outputDirectory", "restore"), () => RunMsBuild("Build"));
 
+targets.Add(
+    "coverity",
+    DependsOn("clean", "outputDirectory", "coverityDirectory", "restore"),
+    () =>
+    {
+        CoverityMsBuild();

```

Comment:
This method is small and this is the only call to it. Would it better to inline it here?